crates/rattler/src/install/installer/mod.rs

```diff
@@ -314,6 +325,9 @@ impl Installer {
         let transaction = Transaction::from_current_and_desired(
             installed,
             records.into_iter().collect::<Vec<_>>(),
+            self.reinstall_packages
```

Comment:
option implements into_iter, so you can do .into_iter().flatten()

Reply:
Oh wait I get it :)